### ISSUE TITLE
Put Trezor logo in the middle of QR code

### ISSUE THREAD

**Issue by mnuky** [trezor/trezor-suite#7177]

**Comment by Hannsek** [comment]
@mnuky Receive address qr code?

**Comment by mnuky** [comment]
yes, still just an idea, we can put small Trezor logo in the middle of QR codes we generate in the app. Small brand touch.

**Comment by Hannsek** [comment]
We should maybe sync with Suite/FW about this. Otherwise good idea. 👍🏻

**Comment by mnuky** [comment]
@Hannsek sure, when we get to it, I will align with you :)
